src/rest-server/src/routes/v2/job.js

```diff
@@ -37,6 +38,18 @@ router
   /** GET /api/v2/jobs/:frameworkName - Get job */
   .get(token.check, controller.get);
 
+router
+  .route('/:frameworkName/attempts/:jobAttemptId')
+  /** GET /api/v2/jobs/:frameworkName/attempts/:jobAttemptId - Get job with specific attempt */
+  .get(token.check, controller.get);
+
+router
+  .route(
+    '/:frameworkName/attempts/:jobAttemptId/taskRoles/:taskRoleName/taskIndex/:taskIndex/attempts',
+  )
```

Comment:
Format?

Reply:
> Format?

There will be  prettier lint error if we put the route in one line.